### PR TITLE
Introduction ⭐ 

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**
Covered the basic introduction to Boostrap framework in a simple and easy-to-understand language.

### CODE REVIEW

File: C-09-bootstrap/01-introduction.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Chapter- 1 Introduction To Bootstrap
+
+Welcome to the course of Bootstrap. In this course we will come to know about the topics like what, how, why related to bootstrap. We will also do practical but in different repository <a href="https://github.com/Vaibhavpratapsingh22/website">click here</a> to see the practical.
+
+## What is Bootstrap?
+
+### "Bootstrap is a free, open-source front-end library for designing websites and web applications. It contains HTML- and CSS-based design templates for everything from typography, forms, buttons, navigation and other interface components as well as JavaScript extensions. Unlike many other web frameworks, Bootstrap concerns itself with front-end development only." - Wikipedia
```
Comment (udeysingh95) [inline C-09-bootstrap/01-introduction.md:7]: why is it written like a header?

File: C-09-bootstrap/01-introduction.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+So basically, bootstrap is an open-source website or library in which there is pre-made codes for different essentials required to built a website. We can customize them easily with basic knowledge of HTML and CSS. There are many components, styles of nav & bars etc.
+
+## How Bootstrap was started?
```
Comment (udeysingh95) [inline C-09-bootstrap/01-introduction.md:11]: this section is giving a lot of extra information. Either make it interesting to read or skip this section in all.

File: C-09-bootstrap/01-introduction.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+Easy to get started: It is easy to get started as it only requires the knowledge of HTML and CSS only.
+Open source: It is open source in nature and completely free to download.
+
+### 3. Great Grid system
+
+Bootstrap makes use of the powerful 12 column grid system which is easy to use and is very consistent.
+
+## Summary
+
+In this chapter we have seen introduction to bootstrap and I hope that you all have tried the starter template to see how it works. In the next chapter we will learn about content system of bootstrap.
```
Comment (udeysingh95) [inline C-09-bootstrap/01-introduction.md:123]: work on the wording here. Also, you are expecting others to have tried something already when you have only mentioned the include the bootstrap. I like the topic and also, I don't know it - so, I can review these PRs as your student aiming to learn bootstrap.
From this chapter, I only learned theory about the bootstrap, to be honest, no application and I can not directly use it as well.

**Review by udeysingh95 — CHANGES_REQUESTED** [review]

File: C-09-bootstrap/01-introduction.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+</body>
+</html>
+```
+
+### 2. Adding properties of CSS only
+
+To add CSS in your code copy-paste the stylesheet <link> into your <head> before all other stylesheets to load our CSS.
+
+```
```
[the remark this answers is not in the document]
Comment (Vaibhavpratapsingh22) [reply C-09-bootstrap/01-introduction.md]: @udeysingh95, these have to like this as if I make them in different lines then when anyone will copy this code they have to reformat it again to make it in a single line. There is a button for copy that will copy the entire code in one-go.

File: C-09-bootstrap/01-introduction.md (hunk 3 above)
Comment (Vaibhavpratapsingh22) [reply C-09-bootstrap/01-introduction.md]: @udeysingh95, in this repo we are just going to understand the Bootstrap theory, the practical part will be done in the second repo. Please check this out by [clicking here](https://github.com/Vaibhavpratapsingh22/website)

**Review by udeysingh95 — APPROVED** [review]

**Pull request merged**